package.json

```diff
@@ -70,23 +70,22 @@
     "version": "manual-git-changelog onversion"
   },
   "devDependencies": {
+    "@types/fs-extra": "^8.0.0",
     "@types/jest": "^29.0.0",
     "@types/minimist": "^1.2.0",
     "@types/n3": "^1.1.4",
+    "@types/node-fetch": "^2.5.0",
     "jest": "^29.0.0",
     "jest-rdf": "^1.3.0",
     "manual-git-changelog": "^1.0.1",
     "nock": "^13.0.0",
     "pre-commit": "^1.2.2",
-    "precommit": "^1.2.2",
     "ts-jest": "^29.0.0",
     "tslint": "^6.0.0",
     "tslint-eslint-rules": "^5.4.0",
     "typescript": "^4.6.2"
   },
   "dependencies": {
-    "@types/fs-extra": "^8.0.0",
```

Comment:
Types belong in deps, not devDeps.

Reply:
Oh, I did not know this belongs there. Thank you.